CommandHelper/LocalPackages/commands/buffs/hayley.ms

```diff
@@ -12,7 +12,9 @@
             'powers': array(
                 "/buff",
                 "/jump 0-5",
-                "/run 0-6"
+                "/run 0-6",
+                "/d wolf",
+                "/fixskin ..which will change you back to normal"
```

Comment:
Give the permission "ch.alias.normalform" then replace this power with /normalform, because thats what other heroes use who need to transform back to normal